third_party/grpc-rust/protoc-rust-grpc/src/lib.rs

```diff
@@ -26,6 +26,8 @@ pub struct Args<'a> {
     pub rust_protobuf: bool,
     /// Customize rust-protobuf codegen
     pub rust_protobuf_customize: protoc_rust::Customize,
+    /// Crate name for oak crate; defaults to "oak" if empty.
+    pub oak_crate: &'a str,
```

Comment:
This feels quite hacky. Also, shouldn't the generated code be part of `oak_abi`, rather? And perhaps the log pseudo-node may also get a similar treatment, i.e. a gRPC server definition (with a single method on it). This does seem to go towards the direction of having the ABI more and more specified via an IDL, similar to how Fuchsia does modules (e.g. https://fuchsia.dev/reference/fidl/fuchsia.logger) -- which I support! But perhaps we should think that through some more and figure out how to best tackle the issue as a whole? ref #426

Reply:
Dropped this and raised a separate issue (#757) for now, to unblock this PR.